src/analytics/redux/tracked_events.js

```diff
@@ -111,20 +221,35 @@ function handleBlockMovedByPosition( clientIds, toIndex ) {
 
 export const trackedEvents = {
 	'core/block-editor': {
-		insertBlock( blocks ) {
-			trackBlocksHandler(
+		insertBlock( blocks, index, rootClientId, updateSelection, meta ) {
+			trackBlockInsertion(
 				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
+				index,
+				rootClientId,
+				updateSelection,
+				undefined,
+				meta
```

Comment:
When I suggested [this](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976#discussion_r1267771581), we had the following code:

https://github.com/wordpress-mobile/gutenberg-mobile/blob/c6163ef83ba8ca7c3a30b03126ef93faefe5c004/src/analytics/redux/tracked_events.js#L197-L198

With this change, seems we can remove all the unused arguments, i.e.:
```
insertBlock( blocks, index, rootClientId, updateSelection, meta ) {
	trackBlockInsertion( blocks, meta );
},
```

This way we could also remove:

https://github.com/wordpress-mobile/gutenberg-mobile/blob/effe376cb59ea829723d0d8538e4bfc8731441db/src/analytics/redux/tracked_events.js#L117-L121

WDYT?

Reply:
Good idea! It'll simplify it a bit, thanks! Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976/commits/94df0273bad2f0dc1a9422b4a590e5ecb7b3beb5